React/Views/ScrollView/RCTScrollableProtocol.h

```diff
@@ -33,7 +33,7 @@
 /**
  * Denotes a view which implements custom pull to refresh functionality.
  */
-@protocol RCTCustomRefreshContolProtocol
+@protocol RCTCustomRefreshControlProtocol
```

Comment:
Oops I was late 😅 saw your comment above.

Reply:
As I said above. I will rename it, but not from one day to another.
I'll let the two protocols live for a couple of versions, so that users can be prompted on updating the naming and then we can remove it in 0.74 or 0.75, for example.

Reply:
@cipolleschi I'm not sure if what you suggested is documented or not, but your suggestion above is _very informative_ at least in the context of renaming public APIs.

Provided typo-fix PRs are frequent, I suggest documenting your suggestions above in the Contributing Guide. Glad to open a PR (on react-native-website) for adding those suggestions. Lmk :)

Reply:
Hi @Pranav-yadav! That would be amazing, although I don't have a clear idea on where it should go.

Probably in the [Contributing](https://reactnative.dev/contributing/overview) section of the Website? 🤔 
To contribute to the website, you can work on the [website repo](https://github.com/facebook/react-native-website). ;)